users/urls.py

```diff
@@ -3,6 +3,7 @@
 
 urlpatterns = [
     path('register/', views.UserRegistrationAPIView.as_view()),
+    path('create_manager/', views.CreateManagerAPIView.as_view()),
```

Comment:
doen't create API and it's endpoint to create the manager role.

Create command for this.